ss_wave_2.rb

```diff
@@ -0,0 +1,219 @@
+
+class SolarSystem
+  attr_accessor :ss
+
+  def initialize(planet_1, planet_2 )
+    @ss = [planet_1, planet_2]
+
+  end
+
+  # adds another planet
+  def add_planet(new_planet)
+    @ss << new_planet
+
+  end
+  # prints the planets out in a list
+  def planet_list
+    list = []
+    count = 0
+    @ss.each do |planets|
+      count += 1
+      list << "#{count}.#{planets.planet_name}\n"
+
+    end
+    return list
+  end
+
+  def pick_planet(user_pick)
+    @ss.each do |planets|
+      if planets.planet_name == user_pick
+        puts "#{planets.details}\n\n\n#{planets.planet_attributes}"
```

Comment:
I love the idea of this method! One way you might make it more flexible is to return the `Planet` itself, rather than `puts`ing a string. That way, the caller could do whatever they like with it.